bot/handlers.py

```diff
@@ -531,16 +483,19 @@ def fallbacks_handler(
 
     def get_conversation_handler(self,) -> ConversationHandler:
         conv_handler = ConversationHandler(
-            entry_points=[MessageHandler(Filters.text, self.expense_name)],
+            entry_points=[MessageHandler(Filters.text, self.expense_name),
+                          CallbackQueryHandler(self.cancel_button_handler)],
             states={
-                States.EXPENSE_SUM: [MessageHandler(Filters.text, self.expense_sum)],
+                States.EXPENSE_SUM: [MessageHandler(Filters.text, self.expense_sum),
+                                     CallbackQueryHandler(self.cancel_button_handler)],
                 States.DEBTOR_NAME: [CallbackQueryHandler(self.debtor_name)],
                 States.DEBT_SUM: [MessageHandler(Filters.text, self.debt_sum)],
             },
             fallbacks=[CallbackQueryHandler(self.fallbacks_button_handler),
                        MessageHandler(Filters.all, self.fallbacks_handler)],
             map_to_parent={
-                ConversationHandler.END: ConversationHandler.END
+                ConversationHandler.END: ConversationHandler.END,
+                States.EVENT_ACTIONS: States.EVENT_ACTIONS
```

Comment:
Please add single comma to the end of the line

Reply:
Done